src/bp/index.ts

```diff
@@ -67,6 +67,11 @@ try {
           description: 'Whether you want to run in production mode or not',
           default: false,
           type: 'boolean'
+        },
+        autoMigrate: {
```

Comment:
wait .. just to be sure, if set to true, Botpress **will** modify the content without notification. right ?